File: apps/frontend/mobile/src/types/enums/Gender.ts
export enum EnumGender {
    M = "M",
    F = "F",
    O = "O",
};

export const GenderLabels: Record<EnumGender, string> = {
    [EnumGender.M]: 'Masculin',
    [EnumGender.F]: 'Féminin',
    [EnumGender.O]: 'Mixte / Autre',
};

File: apps/backend/notification-service/src/main/java/com/blockout/notifications/notification/infrastructure/http/models/Gender.java
package com.blockout.notifications.notification.infrastructure.http.models;

public enum Gender {
    M("Masculin"),
    F("Féminin"),
    O("Mixte / Autre");

    private final String label;

    Gender(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}
